### PR TITLE
Gatsby Image Visualization

### PR DESCRIPTION
**Pull request by laurieontech**
Clean version of #21915 since it was pretty stale.

### CODE REVIEW

File: www/src/assets/icons/layer-icon.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ const LayerIcon = ({ name, fillColor = colors.grey[50] }) => {
         />
       </svg>
     ),
-    Data: (
+    GraphqlLogo: (
```
Comment (AishaBlake) [inline www/src/assets/icons/layer-icon.js:110]: I feel like `GraphQLLogo` makes sense here but by no means should that hold up the PR.

File: www/src/components/layer-model/component-model/component-content-sections.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Link } from "gatsby"
```
Comment (AishaBlake) [inline www/src/components/layer-model/component-model/component-content-sections.js:3]:
```suggestion
```

Looks like this isn't being used anymore?

File: www/src/assets/icons/layer-icon.js (hunk 1 above)
Comment (laurieontech) [reply www/src/assets/icons/layer-icon.js:110]: Maybe GraphQlLogo? Idk, it all seems not ideal!

**Comment by laurieontech** [comment]
So I added lingui for two things. My question there is that translation is a little messy because it affects the code snippets as well. So right now only a piece is translated? That seems worse, open to suggestions!

File: www/src/assets/icons/layer-icon.js (hunk 1 above)
Comment (AishaBlake) [reply www/src/assets/icons/layer-icon.js:110]: Ooh, maybe just leave it alone. It's not terrible! 😅

**Comment by AishaBlake** [comment]
In my ideal world, yeah, the code snippets and _everything_ are translatable. You're right in that it should be all or nothing. Maybe that means extracting those strings out into variables? The paragraphs in Markdown files are fine as they are.

**Comment by laurieontech** [comment]
I might cc: @gillkyle since he has more context on the source array and what it's doing/how it's working. Any ideas on making that translatable?

**Comment by AishaBlake** [comment]
Probably the most important part of this is being able to translate the title of each section.

**Comment by laurieontech** [comment]
No worries. I should have realized! We changed the text to be markdown and I totally spaced on the section titles.

**Comment by gillkyle** [comment]
Oops sorry I got the notification about this, but didn't get to it fast enough. Great to see this get merged in but let me know if there's anything else that should be updated 🙂

**Pull request merged**